install_script.sh.template

```diff
@@ -94,6 +156,9 @@ Troubleshooting and basic usage information for the $nice_flavor are available a
 
     https://docs.datadoghq.com/agent/basic_agent_usage/\n\033[0m\n"
 
+    ERROR_CODE=1
```

Comment:
Could the meaning of each error code be made explicit in a comment somewhere in the file? For instance, after a quick reading of the file, it seems like 6 is for input errors, 5 is for OS / packages not supported, 1 is for all other generic errors, etc. and it would be nice to have a recap of these in the file.

Reply:
Yes you are right they are already defined in this sheet here https://docs.google.com/spreadsheets/d/1GWk23Y5VRw0IK7wdhNMy10OZYTwODasI518deYiMM_Y/edit#gid=0

Reply:
I assigned each code to a variable at the top of the file and used those through instead of numbers